### PR TITLE
[UX2.0] Adds Transport Cellular Controller & Transport Cellular Profile Features

### PR DESCRIPTION
**Pull request by seconroy**
### Description

Adds Transport Cellular Controller & Transport Cellular Profile Features resources and data sources:

### Types of Changes

* [x] New feature (non-breaking change which adds functionaality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Build/CI change
* [ ] Code quality improvement/refactoring/documentation (no functional changes)

### Checklist

* [x] My code follows the code style of this project
* [x] I have added tests to cover my changes
* [x] All new and existing tests pass locally

### CODE REVIEW

File: gen/definitions/profile_parcels/transport_cellular_controller.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+skip_minimum_test: true
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_transport_feature_profile.test.id
+  - model_name: configType
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/transport_cellular_controller.yaml:16]: This field seems to be only in API and not in the UI, also, it looks like in API is mandatory?

File: gen/definitions/profile_parcels/transport_cellular_profile.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+skip_minimum_test: true
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_transport_feature_profile.test.id
+  - model_name: configType
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/transport_cellular_profile.yaml:16]: This field seems to be only in API and not in the UI, also, it looks like in API is mandatory?

File: gen/definitions/profile_parcels/transport_cellular_controller.yaml (hunk 1 above)
Comment (tzarski0) [reply gen/definitions/profile_parcels/transport_cellular_controller.yaml:16]: In cases like this where field is not present in UI, mandatory in API, enum and has only one possible value, maybe we should set it by default it non-eSim (if user doesn't specify something else in the future)?

File: gen/definitions/profile_parcels/transport_cellular_profile.yaml (hunk 2 above)
Comment (tzarski0) [reply gen/definitions/profile_parcels/transport_cellular_profile.yaml:16]: In cases like this where field is not present in UI, mandatory in API, enum and has only one possible value, maybe we should set it by default it non-eSim (if user doesn't specify something else in the future)?

File: gen/definitions/profile_parcels/transport_cellular_controller.yaml (hunk 1 above)
Comment (seconroy) [reply gen/definitions/profile_parcels/transport_cellular_controller.yaml:16]: This is already included by default, is not user definable and does not show in documentation

File: gen/definitions/profile_parcels/transport_cellular_profile.yaml (hunk 2 above)
Comment (seconroy) [reply gen/definitions/profile_parcels/transport_cellular_profile.yaml:16]: This is already included by default, is not user definable and does not show in documentation

**Pull request merged**